### DISCUSSION TITLE
My Editor's default 'data' block is not working when passing a variable to it in the 'OutputData' type.

### DISCUSSION THREAD

**Discussion by Mike-Joseph-Ball** [codex-team/editor.js#2885] [Q&A]
I am attempting to initialize my Editor with previously saved data by passing it through the editor's data property:
![image](https://github.com/user-attachments/assets/73e75f82-2a6e-4fa6-a062-7fbe16a1d623)

My editor remains empty when I pass it as a variable. However, when I try to pass it as a literal 'OutputData' object rather than a variable, it displays properly.
![image](https://github.com/user-attachments/assets/4e90614e-dd23-4a21-a0ec-588dd934392e)
Are you not able to pass variables to the data property? Or am I missing something?

Thank you.

**Comment by idebenone** [comment]
Hi @Mike-Joseph-Ball ,

Please refer to this basic implementation of Editor.js in React + Typescript.
https://codesandbox.io/p/sandbox/editor-js-with-react-typescript-q8p987

If your problem isn't resolved, let us know.

**Reply by Mike-Joseph-Ball** [reply]
Hi @idebenone 

I am not able to access the sandbox. I signed in with the same github account I used here, if you need to grant me viewing permissions. Does this sandbox initialize saved data like I am trying to do? Thank you.
![image](https://github.com/user-attachments/assets/ec016ef4-d510-4772-b1c8-770fe3e6c6fb)

**Reply by idebenone** [reply]
Hi @Mike-Joseph-Ball 

I have made it public. Please check again.
